app/models/sub_section.rb

```diff
@@ -0,0 +1,70 @@
+class SubSection
+  attr_reader :sub_section_slug
+
+  delegate :title, to: :tag_from_content_api
+
+  def initialize(sub_section_slug)
+    @sub_section_slug = sub_section_slug
+  end
+
+  def lists
+    @lists ||= curated_lists || fallback_list_of_uncurated_links
+  end
+
+  def a_to_z?
+    lists
+    @a_to_z
```

Comment:
I think this would be clearer having the logic to work out if it is an a_to_z list in here rather than using a return at the top of curated_lists, so something like

```
def a_to_z?
  !(subsection_from_content_store.details && subsection_from_content_store.details.groups)
end
```

That would also enable to you make the `lists` method slightly easier to read as currently the double `||` is slightly confusing. Something like

```
def lists
  if a_to_z?
    @lists ||= fallback_list_of_uncurated_links
  else
    @lists ||= curated_lists
  end
end
```

It is then very obvious as to what lists is going to contain and why it would contain it.